src/jitlog/reader.lua

```diff
@@ -328,6 +359,8 @@ local function makereader(mixins)
     flushes = {},
     exits = 0,
     gcexits = 0, -- sub count of trace exits forced by the gc state being in atomic or finalize
+    gccycles = 0, -- number GC full cycles that have been seen in the log
```

Comment:
I'm not sure `gccycles` is a great name, because when I see "cycle" and "GC" I think of cyclic data structures. Would "major GCs" be equally accurate?

Reply:
I don't think "major GCs" makes sense since Lua GC's is a two color incremental mark and sweep. Lua's propagate gc state is the mark phase.

Reply:
OK, can you propose something other than "cycle" as a suitable name?

Reply:
Well the code refers to it as a "full GC cycle" https://github.com/softdevteam/LuaJIT/blob/master/src/lj_gc.c#L737

Reply:
Ug, OK. Not very nice terminology, IMHO, but if that's the standard, that's the standard.

Reply:
naming updated